setup.py

```diff
@@ -4,48 +4,69 @@
 with open("README.md", "r") as fh:
     long_description = fh.read()
 
+
+def read_requirements(file):
+    with open(file) as f:
+        return f.read().splitlines()
+
+
 setup(
-    name="spacebench", 
+    name="spacebench",
     version="0.0.1",
-    author=("Mauricio Tec, Ana Trisovic, Audirac, Michelle, Jie Hu," 
-            "Sophie Mirabai Woodward, Naeem Khoshnevis, Francesca Dominici"),
-    author_email=("mauriciogtec@hsph.harvard.edu,"
-                  "anatrisovic@g.harvard.edu,"
-                  "maudirac@hsph.harvard.edu,"
-                  "khu@hsph.harvard.edu,"
-                  "swoodward@fas.harvard.edu,"
-                  "nkhoshnevis@g.harvard.edu,"
-                  "fdominic@hsph.harvard.edu"),
+    author=(
+        "Mauricio Tec, Ana Trisovic, Audirac, Michelle, Jie Hu,"
+        "Sophie Mirabai Woodward, Naeem Khoshnevis, Francesca Dominici"
+    ),
+    author_email=(
+        "mauriciogtec@hsph.harvard.edu,"
+        "anatrisovic@g.harvard.edu,"
+        "maudirac@hsph.harvard.edu,"
+        "khu@hsph.harvard.edu,"
+        "swoodward@fas.harvard.edu,"
+        "nkhoshnevis@g.harvard.edu,"
+        "fdominic@hsph.harvard.edu"
+    ),
     maintainer="Naeem Khoshnevis",
-    maintainer_email = "nkhoshnevis@g.harvard.edu",
-    description= ("spacebench offers benchmark datasets for causal inference, " 
-                  "incorporating real confounder and exposure data from "
-                  "environmental health studies. Synthetic outcomes are "
-                  "generated using machine learning, with spatial confounding "
-                  "introduced by masking key confounders."),
+    maintainer_email="nkhoshnevis@g.harvard.edu",
+    description=(
```

Comment:
The description might be lengthy and get rejected, but I am not sure. If it happens, I can reach out to you.